From 5ba95fa1b60287b4f85cb3fb060b95c5b4de84f4 Mon Sep 17 00:00:00 2001
From: Mamoru TASAKA <mtasaka@fedoraproject.org>
Date: Wed, 28 May 2014 17:52:49 +0900
Subject: [PATCH] po/Makefile.in.in: add comments after POTFILES line

With recent intltool, the next line of POTFILES line is stripped out
without comments, so just add a comment...
---
 po/Makefile.in.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/po/Makefile.in.in b/po/Makefile.in.in
index 8fc246c..28d9313 100644
--- a/po/Makefile.in.in
+++ b/po/Makefile.in.in
@@ -151,6 +151,7 @@ DISTFILES = $(DISTFILES_1) $(GETTEXT_PACKAGE).pot \
 TARFILES = $(DISTFILES_1) $(POFILES) $(SOURCES)
 
 POTFILES = \
+# This comment gets stripped out
 
 CATALOGS = @CATALOGS@
 CATOBJEXT = @CATOBJEXT@
-- 
1.9.3

